qiita_db/test/test_reference.py

```diff
@@ -70,6 +77,60 @@ def test_create(self):
                [tree_id, exp_tree, 12, '0', 1, 6]]
         self.assertEqual(obs, exp)
 
+    def test_create_with_smr_indexed_db(self):
+        fp_count = get_count('qiita.filepath')
+
+        # We need to create a sortmerna db
+        suffixes = ['.bursttrie_0.dat', '.kmer_0.dat', '.pos_0.dat', '.stats']
+        smr_dir = mkdtemp()
```

Comment:
According to [the documentation](https://docs.python.org/2/library/tempfile.html#tempfile.mkdtemp), the creator is responsible for removing the temporary directory.